Configuration.props

```diff
@@ -55,8 +54,8 @@
     <AndroidPreviousFrameworkVersion Condition=" '$(AndroidPreviousFrameworkVersion)' == '' ">v1.0</AndroidPreviousFrameworkVersion>
     <AndroidToolchainCacheDirectory Condition=" '$(AndroidToolchainCacheDirectory)' == '' ">$(HOME)\android-archives</AndroidToolchainCacheDirectory>
     <AndroidToolchainDirectory Condition=" '$(AndroidToolchainDirectory)' == '' ">$(HOME)\android-toolchain</AndroidToolchainDirectory>
-    <AndroidMxeInstallPrefix Condition=" '$(AndroidMxeInstallPrefix)' == '' And '$(NeedMxe)' == 'true' ">$(AndroidToolchainDirectory)\mxe-b9cbb53</AndroidMxeInstallPrefix>
     <AndroidMxeInstallPrefix Condition=" '$(HostOS)' == 'Linux' ">\usr</AndroidMxeInstallPrefix>
+    <AndroidMxeInstallPrefix Condition=" '$(HostOS)' == 'Darwin' ">\usr\local</AndroidMxeInstallPrefix>
```

Comment:
I fear -- but don't know how to verify -- that this may break if Homebrew is *not* located in `/usr/local`.

As it so happens, *I* don't have Homebrew installed in `/usr/local`.

¯\_(ツ)_/¯

Reply:
We'll need to use `brew --prefix` in OS-detection phase to learn the homebrew prefix. I'll push an update soon